README.md

```diff
@@ -280,6 +281,10 @@ as an example, put up a PR with a link! Make sure to use the new [pull request t
 
 - [neondatabase/postgresql_anonymizer](https://github.com/neondatabase/postgresql_anonymizer): A PostgreSQL extension to directly mask or replace Personally Identifiable Information (PII) or commercially sensitive data.
 
+## Financial Ledgers
+
+- [pgledger](https://github.com/pgr0ss/pgledger)
```

Comment:
@pgr0ss: Do you mind filling in the `...` with an easy one-liner? It'll help readers w/ content before needing to click through.

```suggestion
- [pgledger](https://github.com/pgr0ss/pgledger): ...
```

Reply:
How about this? https://github.com/Olshansk/postgres_for_everything/pull/22/commits/b86b541645519795f54d14f393d8e8231f98b2b8